Taskfile.yml

```diff
@@ -157,8 +157,8 @@ tasks:
   nodejs:
     vars:
       NODEJS_VERSION: "14.21.3"
-      TAR_FILE_NAME: "node-v14.21.3-linux-x64.tar.xz"
-      TAR_FILE_PATH: "{{.NODEJS_BUILD_DIR}}/node-v14.21.3-linux-x64.tar.xz"
+      NODEJS_ARCH: "{{ if eq ARCH \"arm64\" }}arm64{{ else }}x64{{ end }}"
```

Comment:
Where is `ARCH` defined?

Reply:
ARCH is a actually a function provided by TaskFile, see https://taskfile.dev/usage/#platform-specific-tasks-and-commands